components/org.wso2.carbon.identity.oauth.endpoint/src/test/java/org/wso2/carbon/identity/oauth/endpoint/authz/OAuth2AuthzEndpointTest.java

```diff
@@ -2497,5 +2497,23 @@ public void testHandleOAuthProblemException(String error, String description, St
         String location = String.valueOf(response.getMetadata().get(HTTPConstants.HEADER_LOCATION).get(0));
         assertEquals(location, expectedUrl);
     }
+    @Test
+    public void testPKCEunsupportedflow() throws Exception {
 
+        OAuth2ClientValidationResponseDTO validationResponseDTO = new OAuth2ClientValidationResponseDTO();
+        OAuthAppDO oAuthAppDO = new OAuthAppDO();
```

Comment:
Lets set the pkceMandatory in the OAuthAppDO to true. So we can make sure regardless of that value the PKCE_UNSUPPORTED_FLOW is given precedence

Reply:
Addressed with https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/2112/commits/031059336c8ebbb1880df10e51f3073578c208b9